packages/jsx-email/test/conditional-raw-minimal.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { jsxToString } from '../src/renderer/jsx-to-string.js';
+import { render } from '../src/renderer/render.js';
+import { Conditional } from '../src/components/conditional.js';
+import { Raw } from '../src/components/raw.js';
+
+// We stub core plugins to keep the bundler from trying to resolve unpublished build outputs
+// during test collection. These stubs are no-ops; we're not testing plugin behavior here.
+vi.mock('@jsx-email/plugin-inline', () => {
+  return {
+    plugin: { name: '@jsx-email/plugin-inline', symbol: Symbol.for('jsx-email/plugin') }
+  };
+});
+vi.mock('@jsx-email/plugin-minify', () => {
+  return {
+    plugin: { name: '@jsx-email/plugin-minify', symbol: Symbol.for('jsx-email/plugin') }
+  };
+});
+vi.mock('@jsx-email/plugin-pretty', () => {
+  return {
+    plugin: { name: '@jsx-email/plugin-pretty', symbol: Symbol.for('jsx-email/plugin') }
+  };
+});
```

Comment:
this is incorrect. you must run the moon build command (e.g. `moon plugin-inline:build`) before running tests locally.